system/bind/templates/etc/_secret_env.tpl

```diff
@@ -0,0 +1,7 @@
+{{- if .Values.secret_env }}
+{{- range .Values.secret_env }}
+{{ .name }}: {{ .value | include "resolve_secret" }}
+{{- end }}
+{{- else }}
+{}
```

Comment:
what's the point of this `else` condition?

Reply:
Aesthetics, mostly. In case there are no secret env vars.
IOW avoid an empty, dangling ```data:``` vs the more explicit ```data: {}```.
Both are valid YAML.